RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# Process for creating a Kube-Router release
+
+## Preparing for the release
+* Ensure that the Golang release used is still supported.
+* Ensure that the Alpine version used for containers is still supported
+* Ensure that Golang dependencies are updated.
+* Ensure that the GoBGP version is updated.
+* Ensure that the Kubernetes object definitions do not contain deprecated object types.
```

Comment:
Maybe you could expound a little more on how to do these items? Basically add links to where they are defined and where they are checked against?

Reply:
Added better explanations